Engineering-Reference/GoogleDestinationAO.md

```diff
@@ -43,71 +43,71 @@ Default Value: Off
 
 Migrated Inbox email directly into 'All Mail' within Google Workspace and not to the Inbox.
 
-Change Conditions: ??
+Change Conditions: Enabling will remove the Inbox label and effectively archive inbox email.
 
 ### Modify Sent Address <a name="modsentadd"></a>
 {: .no_toc }
 Default Value: On
 
 Modify the 'From' header on sent emails to the email address of the destination account if required to enable correct display in Google Workspace.
 
-Change Conditions: ??
+Change Conditions: Disabling will prevent modification of the sent address of migrated emails. This is required for sent mail to be correctly labelled, but can be disabled if performing a domain-switch migration and the destination domain name will be changed to the primary after switchover.
 
 ### Email Import Thread Count <a name="emailthread"></a>
 {: .no_toc }
 Default Value: 50
 
 Set the number of threads that will be used per-user to migrate email to Google Workspace.
 
-Change Conditions: ??
+Change Conditions: The number of email threads can be increased or decreased. Increasing can result in throttling issues and is not recommended.
```

Comment:
remove this part "The number of email threads can be increased or decreased."   What is an email thread?    This is also related to how fast we copy data not just email out of google.      Would say something like "This value alters how fast we pull data from google and should not be altered unless instructed by support as this could result in massive throttling for your migration making it slower.

Reply:
The desciption is above - "he number of threads that will be used per-user to migrate email to Google Workspace.". Agreed, will add a warning.